src/classrooms/classrooms.controller.ts

```diff
@@ -122,4 +122,14 @@ export class ClassroomsController {
     );
     return this.classroomsSerializer.serialize(classroom);
   }
+
+  @Delete(":classroomId/meet-link")
+  @Roles(EUserType.TEACHER)
+  async deleteMeetLink(
+    @Param("classroomId") classroomId: number,
+    @CurrentUser() user: { id: number },
+  ): Promise<ClassroomResponseDto> {
+    const classroom = await this.classroomsService.deleteMeetLink(classroomId, user.id);
+    return this.classroomsSerializer.serialize(classroom);
+  }
```

Comment:
Same comment as previous PR. Can't you just use the update classroom endpoint?

Reply:
Yeah I can, thing is that in the Todo app for marking a task completed I have used a separate endpoint without using update endpoint. So here I thought this would be same